### DISCUSSION TITLE
Embed youtube video

### DISCUSSION THREAD

**Discussion by Had3s99** [requarks/wiki#5170] [Help / Questions]
Hello,

When i edit a page, i can see the youtube video but i can't start it, and when i save my edit the video disappears,

Have u any idea ?

Thanks :)

**Comment by Had3s99** [comment]
Up please someone have any idea ?

**Reply by asisteclick** [reply]
hi! same thing here... I haven't found a solution till now. Have you?

Best,

**Comment by zblcool** [comment]
Hey! check this out. I made it work by following this answer. and you can customize the CSS by yourself

[https://github.com/requarks/wiki/discussions/4580#discussioncomment-1533379](https://github.com/requarks/wiki/discussions/4580#discussioncomment-1533379)

**Reply by Had3s99** [reply]
> Hey! check this out. I made it work by following this answer. and you can customize the CSS by yourself
> 
> [#4580 (comment)](https://github.com/requarks/wiki/discussions/4580#discussioncomment-1533379)

Hey,

Thanks for this answer but even with the edit of css that doesn't work :(

**Reply by Phil-hostwiki** [reply]
> Thanks for this answer but even with the edit of css that doesn't work :(

Did you also insert the corresponding Javascript code?

**Reply by zblcool** [reply]
thats correct. I forgot to mention that you need to include JavaScript as well. The fields should all in the same page

**Reply by Had3s99** [reply]
Ooooooh ok thanks that works !
